app/Models/Hit.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models;
+
+use Carbon\CarbonImmutable;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Hit extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['endpoint', 'user_agent'];
+
+    public static function forTimePeriod(string $period = 'week')
+    {
+        $now = CarbonImmutable::now();
+
+        $currentPeriodHits = Hit::hitsBetween($now->sub($period, 1), $now);
+        $previousPeriodHits = Hit::hitsBetween($now->sub($period, 2), $now->sub($period, 1)->subDay());
+
+        return [
+            'current' => $currentPeriodHits,
+            'previous' => $previousPeriodHits,
+            'change' => $previousPeriodHits
```

Comment:
If change is a percent, which it looks like, we may want to name this `changePercent` or some other word that shows this is a percent instead of a change in count.